### PR TITLE
Update pipelines. Create .editorconfig

### PR DESCRIPTION
**Pull request by ColdForeign**
## Checklist

- [ ] Tests cover new or modified code
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the site documentation
- [ ] I have made corresponding changes to the README, NuGet README file
- [ ] My changes generate no new warnings
- [x] New dependencies added or updated
- [ ] Includes breaking changes
- [ ] Version bumped

### CODE REVIEW

File: .github/linters/.jscpd.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
     "consoleFull"
   ],
   "ignore": [
-    "**/src/Cropper.Blazor/Cropper.Blazor.UnitTests/**"
+	"**/src/Cropper.Blazor/Cropper.Blazor.UnitTests/**",
+	"**/examples/**"
   ],
   "absolute": true
 }
```
Comment (MaxymGorn) [inline .github/linters/.jscpd.json:11]: add end of file

File: .github/workflows/build-test-template.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      working-directory: src/Cropper.Blazor/Cropper.Blazor.UnitTests
+
+    - name: Coverage
+      if: ${{ inputs.publish-coverage == true }}
+      uses: codecov/codecov-action@v4
+      with:
+        token: ${{ secrets.CODECOV_TOKEN }}
+        files: coverage.net6.0.cobertura.xml, coverage.net7.0.cobertura.xml, coverage.net8.0.cobertura.xml
+        fail_ci_if_error: true
+        verbose: true
```
Comment (MaxymGorn) [inline .github/workflows/build-test-template.yml:60]: add end of file

File: .github/linters/.jscpd.json (hunk 1 above)
Comment (ColdForeign) [reply .github/linters/.jscpd.json:11]: in next pr

File: .github/workflows/build-test-template.yml (hunk 2 above)
Comment (ColdForeign) [reply .github/workflows/build-test-template.yml:60]: in next pr

File: .github/linters/.jscpd.json (hunk 1 above)
Comment (MaxymGorn) [inline .github/linters/.jscpd.json:11]:
```suggestion
}

```

File: .github/workflows/build-test-template.yml (hunk 2 above)
Comment (MaxymGorn) [inline .github/workflows/build-test-template.yml:60]:
```suggestion
        verbose: true

```

**Review by MaxymGorn — APPROVED** [review]

**Pull request merged**